migrations/20170731100411-settings-array-to-object.js

```diff
@@ -32,7 +33,7 @@ module.exports = {
       return queryInterface.dropTable('users');
     */
 
-    return qi.changeColumn('RoomType', 'settings', {
+    return qi.changeColumn(TBL, 'settings', {
       type: Sequelize.ARRAY(Sequelize.JSONB),
       allowNull: true,
```

Comment:
We wanted to avoid transforms on responses, and allow json searches, but this is on the down, because thats what we originally did. On the up we switch to JSON